modules/test/playwright/pages/configuration-admin-web/InstanceSettingsPage.ts

```diff
@@ -33,37 +33,47 @@ export class InstanceSettingsPage {
 		await this.page.getByRole('menuitem', {name: 'Export'}).click();
 	}
 
-	async goToInstanceSetting(categoryKey: string, configurationName: string) {
+	async goToInstanceSetting(
+		categoryKey: string,
+		configurationName: string,
+		nthMenuItem?: number
+	) {
 		await this.goto();
+
 		await this.page
 			.getByRole('link', {
 				exact: true,
 				name: categoryKey,
 			})
 			.click();
-		await this.page
-			.getByRole('menuitem', {
-				exact: true,
-				name: configurationName,
-			})
-			.click();
+
+		const configurationMenuItem = this.page.getByRole('menuitem', {
+			exact: true,
+			name: configurationName,
+		});
+
+		if (nthMenuItem) {
+			await configurationMenuItem.nth(nthMenuItem).click();
+		}
+		else {
+			await configurationMenuItem.first().click();
```

Comment:
Using `configurationMenuItem.filter({ hasText: configurationName })` might be safer than using `first()` here.

Reply:
Doesn't both cases return a single element? I don't understands why using the filter would be safe :thinking:

Reply:
I was thinking if the first configurationMenuItem isn't the correct item, it would cause an unintended error.

Reply:
>Also with filter, you could probably remove the nthMenuItem parameter.

@pat270 I added that parameter because in some configurations page, a tab for the site scope is also shown. You can see that when you go to Instance Settings > Pages. There are two `Google PageSpeed Insights`

![image](https://github.com/user-attachments/assets/8bd60e4d-0bbd-458c-8fb6-296a5a996b9e)

Reply:
But you made me look at it differently. I think I can do this
```typescript
	async goToInstanceSetting(
		categoryKey: string,
		configurationName: string,
		nthMenuItem?: number = 0
	) {
            // ...
			await configurationMenuItem.nth(nthMenuItem).click();
```

Reply:
Instead of `nthMenuItem` what about `sectionName`? I noticed Virtual Instance Scope and Site Scope are separate menubars. We can

```
const section = this.page.getByRole('menubar').filter({hasText: sectionName);

const configurationMenuItem = section.getByRole('menuitem', {
    exact: true,
    name: configurationName,
});
```

Reply:
Aaah that's so much better. Thanks @pat270. I will do that one

Reply:
@pat270 bad news. The code doesn't help us. The role "menubar" is not on the ul, but is on a sibling element

![image](https://github.com/user-attachments/assets/4c1d0a01-387c-4d48-8bc5-ca014ca874af)

Reply:
I think you meant

```typescript
this.page.locator("div").filter({hasText: sectionName}).locator('+ div').getByRole('menubar').filter({hasText: configurationName})
```

Reply:
aaah haa! It works
![image](https://github.com/user-attachments/assets/b555191d-5833-47d7-9b39-67424a94093c)

```typescript
const section = this.page
  .locator("div")
  .filter({hasText: sectionName})
  .locator('+ div')
  .getByRole('menubar')

await section.getByRole('menuitem', {
  exact: true,
  name: configurationName,
}).click();
```

Reply:
@pat270 see my changes regarding this in https://github.com/liferay-platform-experience/liferay-portal/pull/752/commits/934324f45a14bc2d5c530bbfaa0a262b0da5d9d7. I am running CI but will be at home by the time it finishes. If you are still here when it finishes, and everything is fine, please forward it, thanks :heart:

Reply:
Nice! @thiago-buarqque sure will do!